test/NuGet.Core.FuncTests/NuGet.XPlat.FuncTest/ListPackageTests.cs

```diff
@@ -19,56 +21,91 @@ public class ListPackageTests
         [Fact]
         public void BasicListPackageParsing_Interactive()
         {
-            // Arrange
-            using (var testDirectory = TestDirectory.Create())
-            {
-                var projectPath = Path.Combine(testDirectory, "project.csproj");
-                File.WriteAllText(projectPath, string.Empty);
+            VerifyCommand(
+                (projectPath, mockCommandRunner, testApp, getLogLevel) =>
+                {
+                    // Arrange
+                    var argList = new List<string> { "list", "--interactive", projectPath };
 
-                var argList = new List<string>() {
-                    "list",
-                    "--interactive",
-                    projectPath};
+                    // Act
+                    var result = testApp.Execute(argList.ToArray());
 
-                var logger = new TestCommandOutputLogger();
-                var testApp = new CommandLineApplication();
-                var mockCommandRunner = new Mock<IListPackageCommandRunner>();
-                mockCommandRunner
-                    .Setup(m => m.ExecuteCommandAsync(It.IsAny<ListPackageArgs>()))
-                    .Returns(Task.CompletedTask);
+                    // Assert
+                    mockCommandRunner.Verify();
+                    Assert.NotNull(HttpHandlerResourceV3.CredentialService);
+                    Assert.Equal(0, result);
+                });
+        }
 
-                testApp.Name = "dotnet nuget_test";
-                ListPackageCommand.Register(testApp,
-                    () => logger,
-                    () => mockCommandRunner.Object);
+        [Fact]
+        public void BasicListPackageParsing_InteractiveTakesNoArguments()
```

Comment:
Test method names should describe what's the test result, as described in the [NuGet coding guidelines](https://github.com/NuGet/NuGet.Client/blob/dev/docs/coding-guidelines.md#unit-test-method-naming)

For example:

```suggestion
        public void BasicListPackageParsing_InteractiveTakesNoArguments_ThrowsException()
```